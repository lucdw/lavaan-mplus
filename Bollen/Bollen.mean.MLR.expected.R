mplus.out <- "Bollen.mean.MLR.expected.out" # needed for batch-execution
library(lavaan)

Data <- read.table("democindus.txt", na.strings = "-999999", 
col.names = c("y1", "y2", "y3", "y4", "y5", "y6", "y7", "y8", "x1", "x2", "x3"))

model <- '
  # measurement model
    dem60 =~ y1 + y2 + y3 + y4
    dem65 =~ y5 + equal("dem60=~y2")*y6
                + equal("dem60=~y3")*y7
                + equal("dem60=~y4")*y8
    ind60 =~ x1 + x2 + x3

  # regressions
    dem60 ~ ind60
    dem65 ~ ind60 + dem60

  # residual correlations
    y1 ~~ y5
    y2 ~~ y4 + y6
    y3 ~~ y7
    y4 ~~ y8
    y6 ~~ y8
'
fit <-  sem (model, data = Data
    , estimator  = "MLR"
    , information  = "expected"
    , meanstructure  = TRUE
    )
summary(fit)  # summary(...): removed if executed in batch
