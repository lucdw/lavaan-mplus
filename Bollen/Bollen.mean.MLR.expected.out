Mplus VERSION 8.2
MUTHEN & MUTHEN
01/17/2024   7:42 AM

INPUT INSTRUCTIONS



      Title:     Bollen's political democracy example
      Data:      File      = democindus.txt;
                 listwise  = on;
      Variable:  Names     = y1 y2 y3 y4 y5 y6 y7 y8 x1 x2 x3;
      Analysis:
      !          model=nomeanstructure;
                 estimator=MLR;
                 information=expected;
                 convergence=0.0000001;
      Model:
                 dem60 by y1 (1);
                 dem60 by y2 (2);
                 dem60 by y3 (3);
                 dem60 by y4 (4);
                 dem65 by y5 (1);
                 dem65 by y6 (2);
                 dem65 by y7 (3);
                 dem65 by y8 (4);
                 ind60 by x1 x2 x3;
                 dem60 on ind60;
                 dem65 on ind60 dem60;
                 y1 y2 y3 y4 y2 y6 pwith y5 y6 y7 y8 y4 y8;
      Output:    standardized;









INPUT READING TERMINATED NORMALLY



Bollen's political democracy example

SUMMARY OF ANALYSIS

Number of groups                                                 1
Number of observations                                          75

Number of dependent variables                                   11
Number of independent variables                                  0
Number of continuous latent variables                            3

Observed dependent variables

  Continuous
   Y1          Y2          Y3          Y4          Y5          Y6
   Y7          Y8          X1          X2          X3

Continuous latent variables
   DEM60       DEM65       IND60


Estimator                                                      MLR
Information matrix                                        EXPECTED
Maximum number of iterations                                  1000
Convergence criterion                                    0.100D-06
Maximum number of steepest descent iterations                   20

Input data file(s)
  democindus.txt

Input data format  FREE



UNIVARIATE SAMPLE STATISTICS


     UNIVARIATE HIGHER-ORDER MOMENT DESCRIPTIVE STATISTICS

         Variable/         Mean/     Skewness/   Minimum/ % with                Percentiles
        Sample Size      Variance    Kurtosis    Maximum  Min/Max      20%/60%    40%/80%    Median

     Y1                    5.465      -0.093       1.250   10.67%       2.500      5.000      5.400
              75.000       6.787      -1.104      10.000    6.67%       6.900      7.500
     Y2                    4.256       0.325       0.000   34.67%       0.000      3.333      3.333
              75.000      15.372      -1.426      10.000   21.33%       4.800     10.000
     Y3                    6.563      -0.606       0.000   10.67%       3.333      6.667      6.667
              75.000      10.621      -0.657      10.000    1.33%       6.667     10.000
     Y4                    4.453       0.120       0.000   22.67%       0.000      3.333      3.333
              75.000      11.069      -1.164      10.000   10.67%       6.667      6.667
     Y5                    5.136      -0.233       0.000    6.67%       2.500      5.000      5.000
              75.000       6.735      -0.718      10.000    2.67%       6.250      7.500
     Y6                    2.978       0.911       0.000   40.00%       0.000      0.000      2.233
              75.000      11.224      -0.400      10.000   10.67%       3.333      6.667
     Y7                    6.196      -0.565       0.000   13.33%       3.333      6.667      6.667
              75.000      10.655      -0.672      10.000   26.67%       6.667     10.000
     Y8                    4.043       0.455       0.000   16.00%       0.368      3.333      3.333
              75.000      10.393      -0.906      10.000   12.00%       3.333      6.667
     X1                    5.054       0.259       3.784    1.33%       4.317      4.727      5.075
              75.000       0.530      -0.693       6.737    1.33%       5.226      5.631
     X2                    4.792      -0.353       1.386    1.33%       3.466      4.595      4.963
              75.000       2.252      -0.505       7.872    1.33%       5.236      6.203
     X3                    3.558       0.086       1.002    1.33%       2.113      3.167      3.568
              75.000       1.950      -0.880       6.425    1.33%       3.977      4.586


THE MODEL ESTIMATION TERMINATED NORMALLY



MODEL FIT INFORMATION

Number of Free Parameters                       39

Loglikelihood

          H0 Value                       -1548.818
          H0 Scaling Correction Factor      1.0266
            for MLR
          H1 Value                       -1528.728
          H1 Scaling Correction Factor      0.9690
            for MLR

Information Criteria

          Akaike (AIC)                    3175.636
          Bayesian (BIC)                  3266.018
          Sample-Size Adjusted BIC        3143.100
            (n* = (n + 2) / 24)

Chi-Square Test of Model Fit

          Value                             44.155*
          Degrees of Freedom                    38
          P-Value                           0.2276
          Scaling Correction Factor         0.9100
            for MLR

*   The chi-square value for MLM, MLMV, MLR, ULSMV, WLSM and WLSMV cannot be used
    for chi-square difference testing in the regular way.  MLM, MLR and WLSM
    chi-square difference testing is described on the Mplus website.  MLMV, WLSMV,
    and ULSMV difference testing is done using the DIFFTEST option.

RMSEA (Root Mean Square Error Of Approximation)

          Estimate                           0.046
          90 Percent C.I.                    0.000  0.097
          Probability RMSEA <= .05           0.508

CFI/TLI

          CFI                                0.990
          TLI                                0.986

Chi-Square Test of Model Fit for the Baseline Model

          Value                            702.841
          Degrees of Freedom                    55
          P-Value                           0.0000

SRMR (Standardized Root Mean Square Residual)

          Value                              0.048



MODEL RESULTS

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 DEM60    BY
    Y1                 1.000      0.000    999.000    999.000
    Y2                 1.191      0.133      8.956      0.000
    Y3                 1.175      0.118      9.932      0.000
    Y4                 1.251      0.121     10.353      0.000

 DEM65    BY
    Y5                 1.000      0.000    999.000    999.000
    Y6                 1.191      0.133      8.956      0.000
    Y7                 1.175      0.118      9.932      0.000
    Y8                 1.251      0.121     10.353      0.000

 IND60    BY
    X1                 1.000      0.000    999.000    999.000
    X2                 2.180      0.143     15.204      0.000
    X3                 1.818      0.139     13.072      0.000

 DEM60    ON
    IND60              1.471      0.351      4.196      0.000

 DEM65    ON
    IND60              0.600      0.211      2.846      0.004
    DEM60              0.865      0.062     13.874      0.000

 Y1       WITH
    Y5                 0.583      0.420      1.387      0.166

 Y2       WITH
    Y6                 2.183      0.881      2.478      0.013
    Y4                 1.440      0.699      2.060      0.039

 Y3       WITH
    Y7                 0.712      0.586      1.215      0.224

 Y4       WITH
    Y8                 0.363      0.398      0.910      0.363

 Y6       WITH
    Y8                 1.372      0.738      1.858      0.063

 Intercepts
    Y1                 5.465      0.301     18.166      0.000
    Y2                 4.256      0.453      9.402      0.000
    Y3                 6.563      0.376     17.441      0.000
    Y4                 4.453      0.384     11.590      0.000
    Y5                 5.136      0.300     17.140      0.000
    Y6                 2.978      0.387      7.698      0.000
    Y7                 6.196      0.377     16.439      0.000
    Y8                 4.043      0.372     10.862      0.000
    X1                 5.054      0.084     60.127      0.000
    X2                 4.792      0.173     27.657      0.000
    X3                 3.558      0.161     22.066      0.000

 Variances
    IND60              0.449      0.072      6.190      0.000

 Residual Variances
    Y1                 1.855      0.399      4.644      0.000
    Y2                 7.581      1.227      6.177      0.000
    Y3                 4.956      1.008      4.918      0.000
    Y4                 3.224      0.741      4.353      0.000
    Y5                 2.313      0.571      4.052      0.000
    Y6                 4.968      0.992      5.009      0.000
    Y7                 3.560      0.575      6.191      0.000
    Y8                 3.308      0.842      3.929      0.000
    X1                 0.081      0.018      4.521      0.000
    X2                 0.120      0.073      1.658      0.097
    X3                 0.467      0.086      5.443      0.000
    DEM60              3.875      0.817      4.744      0.000
    DEM65              0.164      0.238      0.691      0.489


STANDARDIZED MODEL RESULTS


STDYX Standardization

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 DEM60    BY
    Y1                 0.850      0.040     21.293      0.000
    Y2                 0.690      0.059     11.781      0.000
    Y3                 0.758      0.050     15.034      0.000
    Y4                 0.838      0.042     19.887      0.000

 DEM65    BY
    Y5                 0.817      0.051     16.078      0.000
    Y6                 0.755      0.052     14.492      0.000
    Y7                 0.802      0.034     23.623      0.000
    Y8                 0.829      0.048     17.431      0.000

 IND60    BY
    X1                 0.920      0.022     42.557      0.000
    X2                 0.973      0.016     59.088      0.000
    X3                 0.872      0.030     29.413      0.000

 DEM60    ON
    IND60              0.448      0.109      4.117      0.000

 DEM65    ON
    IND60              0.187      0.063      2.943      0.003
    DEM60              0.884      0.050     17.789      0.000

 Y1       WITH
    Y5                 0.281      0.167      1.684      0.092

 Y2       WITH
    Y6                 0.356      0.131      2.705      0.007
    Y4                 0.291      0.114      2.564      0.010

 Y3       WITH
    Y7                 0.169      0.138      1.229      0.219

 Y4       WITH
    Y8                 0.111      0.117      0.946      0.344

 Y6       WITH
    Y8                 0.338      0.134      2.519      0.012

 Intercepts
    Y1                 2.111      0.175     12.084      0.000
    Y2                 1.120      0.103     10.877      0.000
    Y3                 1.924      0.196      9.825      0.000
    Y4                 1.354      0.130     10.394      0.000
    Y5                 1.948      0.168     11.606      0.000
    Y6                 0.876      0.085     10.356      0.000
    Y7                 1.963      0.215      9.131      0.000
    Y8                 1.244      0.112     11.070      0.000
    X1                 6.943      0.447     15.548      0.000
    X2                 3.194      0.281     11.349      0.000
    X3                 2.548      0.186     13.685      0.000

 Variances
    IND60              1.000      0.000    999.000    999.000

 Residual Variances
    Y1                 0.277      0.068      4.074      0.000
    Y2                 0.525      0.081      6.498      0.000
    Y3                 0.426      0.076      5.572      0.000
    Y4                 0.298      0.071      4.227      0.000
    Y5                 0.333      0.083      4.009      0.000
    Y6                 0.430      0.079      5.473      0.000
    Y7                 0.357      0.054      6.571      0.000
    Y8                 0.313      0.079      3.971      0.000
    X1                 0.154      0.040      3.860      0.000
    X2                 0.053      0.032      1.669      0.095
    X3                 0.239      0.052      4.628      0.000
    DEM60              0.800      0.097      8.215      0.000
    DEM65              0.035      0.052      0.687      0.492


STDY Standardization

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 DEM60    BY
    Y1                 0.850      0.040     21.293      0.000
    Y2                 0.690      0.059     11.781      0.000
    Y3                 0.758      0.050     15.034      0.000
    Y4                 0.838      0.042     19.887      0.000

 DEM65    BY
    Y5                 0.817      0.051     16.078      0.000
    Y6                 0.755      0.052     14.492      0.000
    Y7                 0.802      0.034     23.623      0.000
    Y8                 0.829      0.048     17.431      0.000

 IND60    BY
    X1                 0.920      0.022     42.557      0.000
    X2                 0.973      0.016     59.088      0.000
    X3                 0.872      0.030     29.413      0.000

 DEM60    ON
    IND60              0.448      0.109      4.117      0.000

 DEM65    ON
    IND60              0.187      0.063      2.943      0.003
    DEM60              0.884      0.050     17.789      0.000

 Y1       WITH
    Y5                 0.281      0.167      1.684      0.092

 Y2       WITH
    Y6                 0.356      0.131      2.705      0.007
    Y4                 0.291      0.114      2.564      0.010

 Y3       WITH
    Y7                 0.169      0.138      1.229      0.219

 Y4       WITH
    Y8                 0.111      0.117      0.946      0.344

 Y6       WITH
    Y8                 0.338      0.134      2.519      0.012

 Intercepts
    Y1                 2.111      0.175     12.084      0.000
    Y2                 1.120      0.103     10.877      0.000
    Y3                 1.924      0.196      9.825      0.000
    Y4                 1.354      0.130     10.394      0.000
    Y5                 1.948      0.168     11.606      0.000
    Y6                 0.876      0.085     10.356      0.000
    Y7                 1.963      0.215      9.131      0.000
    Y8                 1.244      0.112     11.070      0.000
    X1                 6.943      0.447     15.548      0.000
    X2                 3.194      0.281     11.349      0.000
    X3                 2.548      0.186     13.685      0.000

 Variances
    IND60              1.000      0.000    999.000    999.000

 Residual Variances
    Y1                 0.277      0.068      4.074      0.000
    Y2                 0.525      0.081      6.498      0.000
    Y3                 0.426      0.076      5.572      0.000
    Y4                 0.298      0.071      4.227      0.000
    Y5                 0.333      0.083      4.009      0.000
    Y6                 0.430      0.079      5.473      0.000
    Y7                 0.357      0.054      6.571      0.000
    Y8                 0.313      0.079      3.971      0.000
    X1                 0.154      0.040      3.860      0.000
    X2                 0.053      0.032      1.669      0.095
    X3                 0.239      0.052      4.628      0.000
    DEM60              0.800      0.097      8.215      0.000
    DEM65              0.035      0.052      0.687      0.492


STD Standardization

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 DEM60    BY
    Y1                 2.201      0.194     11.351      0.000
    Y2                 2.621      0.283      9.261      0.000
    Y3                 2.586      0.246     10.490      0.000
    Y4                 2.754      0.229     12.030      0.000

 DEM65    BY
    Y5                 2.154      0.200     10.788      0.000
    Y6                 2.565      0.277      9.261      0.000
    Y7                 2.530      0.233     10.870      0.000
    Y8                 2.694      0.249     10.816      0.000

 IND60    BY
    X1                 0.670      0.054     12.380      0.000
    X2                 1.460      0.108     13.508      0.000
    X3                 1.218      0.102     11.882      0.000

 DEM60    ON
    IND60              0.448      0.109      4.117      0.000

 DEM65    ON
    IND60              0.187      0.063      2.943      0.003
    DEM60              0.884      0.050     17.789      0.000

 Y1       WITH
    Y5                 0.583      0.420      1.387      0.166

 Y2       WITH
    Y6                 2.183      0.881      2.478      0.013
    Y4                 1.440      0.699      2.060      0.039

 Y3       WITH
    Y7                 0.712      0.586      1.215      0.224

 Y4       WITH
    Y8                 0.363      0.398      0.910      0.363

 Y6       WITH
    Y8                 1.372      0.738      1.858      0.063

 Intercepts
    Y1                 5.465      0.301     18.166      0.000
    Y2                 4.256      0.453      9.402      0.000
    Y3                 6.563      0.376     17.441      0.000
    Y4                 4.453      0.384     11.590      0.000
    Y5                 5.136      0.300     17.140      0.000
    Y6                 2.978      0.387      7.698      0.000
    Y7                 6.196      0.377     16.439      0.000
    Y8                 4.043      0.372     10.862      0.000
    X1                 5.054      0.084     60.127      0.000
    X2                 4.792      0.173     27.657      0.000
    X3                 3.558      0.161     22.066      0.000

 Variances
    IND60              1.000      0.000    999.000    999.000

 Residual Variances
    Y1                 1.855      0.399      4.644      0.000
    Y2                 7.581      1.227      6.177      0.000
    Y3                 4.956      1.008      4.918      0.000
    Y4                 3.224      0.741      4.353      0.000
    Y5                 2.313      0.571      4.052      0.000
    Y6                 4.968      0.992      5.009      0.000
    Y7                 3.560      0.575      6.191      0.000
    Y8                 3.308      0.842      3.929      0.000
    X1                 0.081      0.018      4.521      0.000
    X2                 0.120      0.073      1.658      0.097
    X3                 0.467      0.086      5.443      0.000
    DEM60              0.800      0.097      8.215      0.000
    DEM65              0.035      0.052      0.687      0.492


R-SQUARE

    Observed                                        Two-Tailed
    Variable        Estimate       S.E.  Est./S.E.    P-Value

    Y1                 0.723      0.068     10.647      0.000
    Y2                 0.475      0.081      5.891      0.000
    Y3                 0.574      0.076      7.517      0.000
    Y4                 0.702      0.071      9.943      0.000
    Y5                 0.667      0.083      8.039      0.000
    Y6                 0.570      0.079      7.246      0.000
    Y7                 0.643      0.054     11.812      0.000
    Y8                 0.687      0.079      8.715      0.000
    X1                 0.846      0.040     21.278      0.000
    X2                 0.947      0.032     29.544      0.000
    X3                 0.761      0.052     14.706      0.000

     Latent                                         Two-Tailed
    Variable        Estimate       S.E.  Est./S.E.    P-Value

    DEM60              0.200      0.097      2.059      0.040
    DEM65              0.965      0.052     18.693      0.000


QUALITY OF NUMERICAL RESULTS

     Condition Number for the Information Matrix              0.306E-03
       (ratio of smallest to largest eigenvalue)


     Beginning Time:  07:42:26
        Ending Time:  07:42:26
       Elapsed Time:  00:00:00



MUTHEN & MUTHEN
3463 Stoner Ave.
Los Angeles, CA  90066

Tel: (310) 391-9971
Fax: (310) 391-8971
Web: www.StatModel.com
Support: Support@StatModel.com

Copyright (c) 1998-2018 Muthen & Muthen
